hydra-node/src/Hydra/Chain.hs

```diff
@@ -129,6 +129,8 @@ data PostTxError tx
     -- NOTE: PlutusDebugInfo does not have much available instances so we put it
     -- in Text form but it's lame
     PlutusValidationFailed {plutusFailure :: Text, plutusDebugInfo :: Text}
+  | -- | User tried to commit more than 100 ADA hardcoded limit on mainnet
```

Comment:
Should: not hard-code the 100 ADA in multiple places.

How much will change over time, don't put it into the haddocks, but into a variable and refer it here.